concepts/60 Themes and Styles/05 Predefined Themes/60 Switch Between Themes at Runtime/7 Switch Between Color Swatches.md

```diff
@@ -0,0 +1,174 @@
+You can use <a href="https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet" target="_blank">StyleSheet API</a> to switch between color swatches. In the following code, theme swatches are called `theme.custom-scheme.light.css` and `theme.custom-scheme.dark.css` and are placed in the same folder as `index.html`. The code uses SelectBox to implement the swatch selector. To switch between theme and their swatches at the same time, call [DevExpress.ui.themes.current(themeName)](/api-reference/50%20Common/utils/ui/themes/current(themeName).md '/Documentation/ApiReference/Common/utils/ui/themes/#currentthemeName') method in the `onValueChanged` callback.
```

Comment:
```suggestion
You can use <a href="https://developer.mozilla.org/en-US/docs/Web/API/StyleSheet" target="_blank">StyleSheet API</a> to switch between color swatches. In the following code, theme swatches are called `theme.custom-scheme.light.css` and `theme.custom-scheme.dark.css` and are placed in the same folder as `index.html`. The code uses SelectBox to implement the swatch selector. To switch between the theme and its swatches at the same time, call [DevExpress.ui.themes.current(themeName)](/api-reference/50%20Common/utils/ui/themes/current(themeName).md '/Documentation/ApiReference/Common/utils/ui/themes/#currentthemeName') method in the `onValueChanged` callback.
```
` at the same time` How can you switch between something 'at the same time'?

Reply:
I meant that you can switch between 2 themes, between 2 swatches, and between theme+swatch and another theme+swatch (the latter is the most common scenario). I know I messed up here, but I don't know how to formulate it clearly

Reply:
Perhaps "To switch between different themes and swatches"?